tests/unit/test-class-wc-payments-utils.php

```diff
@@ -555,4 +555,53 @@ public function test_get_filtered_error_status_code_with_api_exception() {
 	public function test_get_filtered_error_status_code_with_api_exception_and_402_status() {
 		$this->assertSame( 400, WC_Payments_Utils::get_filtered_error_status_code( new \WCPay\Exceptions\API_Exception( 'Error: Your card was declined.', 'card_declined', 402 ) ) );
 	}
+
+	public function delete_appearance_theme_transients( $transients ) {
+		foreach ( $transients as $location => $contexts ) {
+			foreach ( $contexts as $context => $transient ) {
+				delete_transient( $transient );
+			}
+		}
+	}
+
+	public function set_appearance_theme_transients( $transients ) {
+		foreach ( $transients as $location => $contexts ) {
+			foreach ( $contexts as $context => $transient ) {
+				set_transient( $transient, $location . '_' . $context . '_value', DAY_IN_SECONDS );
+			}
+		}
+	}
```

Comment:
```suggestion
	private function delete_appearance_theme_transients( $transients ) {
		foreach ( $transients as $location => $contexts ) {
			foreach ( $contexts as $context => $transient ) {
				delete_transient( $transient );
			}
		}
	}

	private function set_appearance_theme_transients( $transients ) {
		foreach ( $transients as $location => $contexts ) {
			foreach ( $contexts as $context => $transient ) {
				set_transient( $transient, $location . '_' . $context . '_value', DAY_IN_SECONDS );
			}
		}
	}
```
Not really a big deal, but seems like better form to keep utility test functions `private` to help differentiate them from the other tests. 🤷